org.eclipse.xtext.ui.codemining/pom.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
how can me make sure this is built?

Reply:
This will pick for this plugin always the photon target, even when the build is performed with the luna profile. It will override the target-platform-configuration from parent.
This bundle requires photon to be build.

Reply:
Actually we tested that :)

Reply:
or do we always build against photon?

Reply:
No, we can have further multi-platform builds. Only this plugin is build differently. It works fine.